cleanup_utils/dns_cleanup.py

```diff
@@ -173,3 +175,28 @@ def run_cleanup(self, hashes_to_clean):
                         zone_id=self.zone_id,
                         zone_version_id=new_zone_version
                     )
+
+
+class DNSCleanupInstance:
```

Comment:
Nit: `DNSCleanupInstance` -> `DnsCleanupInstance`.
You can keep the old one as `OldDnsCleanupInstance`.